environment.yml

```diff
@@ -139,99 +136,89 @@ dependencies:
-    - backports.shutil-which==3.5.2
+    - backports-shutil-get-terminal-size==1.0.0
+    - backports-shutil-which==3.5.2
     - bleach==3.1.0
     - chardet==3.0.4
-    - cloudpickle==0.6.1
+    - cloudpickle==0.8.1
     - colorama==0.4.1
-    - configparser==3.5.0
-    - cvxopt==1.2.2
-    - dash==0.35.2
-    - dash-core-components==0.42.1
-    - dash-html-components==0.13.5
-    - dash-renderer==0.16.2
-    - decorator==4.3.0
+    - configparser==3.7.4
+    - cvxopt==1.2.3
+    - decorator==4.4.0
     - defusedxml==0.5.0
     - docutils==0.14
     - doit==0.29.0
     - entrypoints==0.3
-    - flask==1.0.2
-    - flask-compress==1.4.0
+    - ephem==3.7.6.0
     - idna==2.8
     - imagesize==1.1.0
+    - intel-openmp==2019.0
     - ipaddress==1.0.22
     - ipykernel==4.10.0
     - ipython==5.8.0
     - ipython-genutils==0.2.0
     - ipywidgets==7.4.2
-    - itsdangerous==1.1.0
     - jinja2==2.10
-    - jsonschema
+    - jsonschema==3.0.1
     - jupyter==1.0.0
     - jupyter-client==5.2.4
     - jupyter-console==5.2.0
     - jupyter-core==4.4.0
-    - lxml==4.3.0
+    - lxml==4.3.3
+    - mako==1.0.7
     - mistune==0.8.4
-    - nbconvert==5.4.0
+    - mkl==2019.0
+    - mock==2.0.0
+    - nbconvert==5.4.1
     - nbformat==4.4.0
     - networkx==2.2
-    - notebook==5.7.4
-    - packaging==18.0
+    - notebook==5.7.7
+    - packaging==19.0
     - pandocfilters==1.4.2
     - pathlib2==2.3.3
+    - pbr==5.1.3
     - pickleshare==0.7.5
     - pkginfo==1.5.0.1
-    - plotly==2.5.0
-    - prometheus-client==0.5.0
+    - plotly==3.7.1
+    - prometheus-client==0.6.0
     - prompt-toolkit==1.0.15
-    - py4design==0.25
+    - py4design==0.26
     - pycollada==0.6
     - pygments==2.3.1
     - pymf==0.1.9
-    - pyshp==2.0.1
+    - pyrsistent==0.14.11
+    - pyshp==2.1.0
     - pywinpty==0.5.5
-    - pyyaml==3.13
-    - pyzmq==17.1.2
+    - pyyaml==5.1
+    - pyzmq==18.0.1
     - qtconsole==4.4.3
     - readme-renderer==24.0
     - requests==2.21.0
-    - requests-toolbelt==0.8.0
+    - requests-toolbelt==0.9.1
     - retrying==1.3.3
     - salib==1.2
-    - scandir==1.9.0
-    - scikit-learn==0.20.2
+    - scandir==1.10.0
+    - scikit-learn==0.20.3
     - seaborn==0.9.0
     - send2trash==1.5.0
     - simplegeneric==0.8.1
     - snowballstemmer==1.2.1
-    - sphinx==1.8.3
+    - sphinx==1.8.5
     - sphinxcontrib-websupport==1.1.0
     - terminado==0.8.1
     - testpath==0.4.2
     - traitlets==4.3.2
-    - twine==1.12.1
+    - twine==1.13.0
     - typing==3.6.6
     - urllib3==1.24.1
     - utm==0.4.2
     - wcwidth==0.1.7
     - webencodings==0.5.1
-    - werkzeug==0.14.1
     - widgetsnbextension==3.4.2
     - win-unicode-console==0.5
     - xlrd==1.2.0
     - xlwt==1.3.0
-    - mpmath==0.19
-    - sympy==1.3
-    - ephem==3.7.6.0
-    - timezonefinder==3.4.2
-    - deap==1.2.2
-    - CoolProp==6.2.1
-    - Pyomo==5.6.1
-    - SQLalchemy==0.9
-    - tables==3.4.4
-    - simpledbf==0.2.6
-    - pvlib==0.5.2
```

Comment:
@JIMENOFONSECA I'm not sure... they might be installed as part of another package. What I did do was create a new environment based on this file and then run `cea test` - that worked. Also, the `INSTALL_REQUIRES` in `setup.py` is the more important place to put this stuff: That is what would be `pip install`ed, say on other platforms like Mac OSX etc. (this is in the pip install part anyway, right?)

I suggest we merge it like this and update on a case-by-case basis when something doesn't work. What do you think?

Reply:
I see,

From my side, I am sure all of these libraries are used, and the version of each library needs to remain to make CEA work.

deap, and pyomo, for example are installed independently and are quite important packages.

I would avoid at all costs to erase libraries, since this will take us back to the problem we had a couple of months ago, when some libraries’ versions entered in conflict with others.

Reply:
Plus, the cea test does not test plots, optimization and beyond. So removing libraries will just brake CEA. If you do not mind could we make any changes to libraries a separate issue?, hence, revert this change here?

Reply:
@JIMENOFONSECA sure! btw: could you please send me your version of `conda env export > environemnt_for_daren.yml` so I can cross-reference? does your environment run everything? (we have multiple versions around and I'd like to avoid making mistakes.

Reply:
@JIMENOFONSECA DONE! `environment.yml` is now identical to master.